pkg/multiclustermanager/routes.go

```diff
@@ -97,12 +98,7 @@ func router(ctx context.Context, localClusterEnabled bool, tunnelAuthorizer *mcm
 
 	authed.Path("/meta/aksVersions").Handler(capabilities.NewAKSVersionsHandler())
 	authed.Path("/meta/aksVirtualNetworks").Handler(capabilities.NewAKSVirtualNetworksHandler())
-	authed.Path("/meta/gkeMachineTypes").Handler(capabilities.NewGKEMachineTypesHandler())
-	authed.Path("/meta/gkeNetworks").Handler(capabilities.NewGKENetworksHandler())
-	authed.Path("/meta/gkeServiceAccounts").Handler(capabilities.NewGKEServiceAccountsHandler())
-	authed.Path("/meta/gkeSubnetworks").Handler(capabilities.NewGKESubnetworksHandler())
-	authed.Path("/meta/gkeVersions").Handler(capabilities.NewGKEVersionsHandler())
-	authed.Path("/meta/gkeZones").Handler(capabilities.NewGKEZonesHandler())
+	authed.Path("/meta/{resource:gke.+}").Handler(gke.NewGKEHandler(scaledContext))
```

Comment:
We need to guarantee that v1 requests haven't changed at all.

Reply:
URL requests were not changed fro v1, double checked